collections/vegetation-indices.yaml

```diff
@@ -15,7 +15,7 @@ EOBrowser: https://apps.sentinel-hub.com/eo-browser/?zoom=10&lat=41.9&lng=12.5&t
 Image: vegetation-indices/thumbnail.png
 Resolution: 10m
 GeographicalCoverage: Europe (Lat; 26N 72N Lon; -25W 45E)
-TemporalAvailability: 2016-10-01 - 2022-11-09
+TemporalAvailability: 2016-10-01 - Ongoing
```

Comment:
We are not 100% consistent here but most other collections that use this have written "ongoing" with small "o". Let's change this for consistency here as well.

Reply:
I noted that - changed , to keep in mind in future entries